assets/src/components/RangeSlider.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import 'rc-slider/assets/index.css'
-import Slider, { Range } from 'rc-slider'
```

Comment:
rc-slider package has some solid change, but not breaking. I felt the slider looked ok and functioned as before